client/pages/quizzes.js

```diff
@@ -2,14 +2,98 @@
   This page will load at the url "/quizzes"
 */
 
-import React from "react";
-import Header from "../components/header/Header";
+import React, { useState, useEffect } from 'react';
+import Header from '../components/header/Header';
+import QuizTile from '../components/quizSelection/QuizTile';
+import TopicSelection from '../components/quizSelection/TopicSelection';
+import { QuizzesHeader, TileSection } from '../components/quizSelection/styles';
+import { PageHeader } from '../components/shared/styles';
 
 export default function Quizzes() {
+  const [chosenSubject, setChosenSubject] = useState("Programming");
```

Comment:
@cMikolai Do we want to pre-select topics?

Reply:
@Soreg Mmm, nope, I would say not to. What do you think?

Reply:
@cMikolai Yeah same. @ZumDeWald Default should probably be empty string :)

Reply:
@Soreg @cMikolai I chose to pre-select Programming as it would likely be the most chosen, if we don't pre-select one we should have something in the area where the quiz tiles would be indicating the user should select from the options.

Reply:
Fixed in the latest push.
For now there is only dummy data, so it starts with "Random" and shows all the dummy quiz tiles.

Once this is closed I will begin adding real routes and will get a random sorting going for all quizzes on page load.